modules/react/modal/lib/ModalOverlay.tsx

```diff
@@ -118,3 +99,12 @@ const OpenModalOverlay = createSubcomponent('div')({
     return null;
   }
 });
+
+export const ModalOverlay = createSubcomponent('div')({
+  displayName: 'Modal.Overlay',
+  modelHook: useModalModel,
+})<ModalOverlayProps>((elemProps, Element, model) => {
+  return model.state.visibility !== 'hidden' ? (
+    <OpenModalOverlay as={Element} model={model} {...elemProps} />
+  ) : null;
```

Comment:
```suggestion
  return model.state.visibility !== 'hidden' && (
    <OpenModalOverlay as={Element} model={model} {...elemProps} />
  );
```

can it be like this?

Reply:
The types are set to `null`. So it has to be `null`